umpleonline/umple.php

```diff
@@ -241,7 +241,7 @@
     &nbsp;
 
     <span style="font-size: 30%; white-space:nowrap;">
-    <a class="button2" href="javascript:Page.toggleTabs()" title="Toggle tab visibility">Tabs</a>
+    <a class="button2 active" href="javascript:Page.toggleTabs()" title="Toggle tab visibility">Tabs</a>
```

Comment:
This makes the button active. But it does not switch between active and inactive appearance as it should between clicks.